Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -1683,48 +1647,53 @@ public void identifyImage(int pageNumber) {
-                logger.error(e);
-            }
-            /* das neue Bild zuweisen */
-            if (this.image != null) {
-                try {
-                    URI tifFile = this.currentTifFolder.resolve(this.image);
-                    logger.trace("tiffconverterpfad: {}", tifFile);
-                    if (!fileService.fileExist(tifFile)) {
-                        tifFile = serviceManager.getProcessService().getImagesTifDirectory(true, this.process)
-                                .resolve(this.image);
-                        Helper.setFehlerMeldung("formularOrdner:TifFolders", "", "image " + this.image
-                                + " does not exist in folder " + this.currentTifFolder + ", using image from "
+                URI tifFile = this.currentTifFolder.resolve(this.image);
+                logger.trace("tiffconverterpfad: {}", tifFile);
+                if (!fileService.fileExist(tifFile)) {
+                    tifFile = serviceManager.getProcessService().getImagesTifDirectory(true, this.process)
+                            .resolve(this.image);
+                    Helper.setFehlerMeldung("formularOrdner:TifFolders", "",
+                        "image " + this.image + " does not exist in folder " + this.currentTifFolder
+                                + ", using image from "
                                 + new File(serviceManager.getProcessService().getImagesTifDirectory(true, this.process))
                                         .getName());
-                    }
+                }
 
-                    // Copy tif-file to temporay folder
-                    InputStream tifFileInputStream = fileService.read(tifFile);
-                    if (temporaryTifFile != null) {
-                        FileUtils.copyInputStreamToFile(tifFileInputStream, temporaryTifFile);
-                        this.imageHelper.scaleFile(temporaryTifFile.toURI(), pagesDirectory.resolve(currentPngFile),
-                            this.imageSize, this.imageRotation);
-                        logger.trace("scaleFile");
-                    }
-                } catch (Exception e) {
-                    Helper.setFehlerMeldung("could not getById image folder", e);
-                    logger.error(e);
-                } finally {
-                    if (temporaryTifFile != null) {
-                        try {
-                            if (!fileService.delete(temporaryTifFile.toURI())) {
-                                logger.error(
-                                    "Error while deleting temporary tif file: " + temporaryTifFile.getAbsolutePath());
-                            }
-                            // not working
-                        } catch (IOException e) {
-                            logger.error("Error while deleting temporary tif file: " + e.getMessage());
+                // Copy tif-file to temporary folder
+                InputStream tifFileInputStream = fileService.read(tifFile);
```

Comment:
Where is the stream closed? Maybe you should use a try with resources statement. You are only moved the code but we should fix it when we touch the code.